cmd/root.go

```diff
@@ -13,5 +19,47 @@ var rootCmd = &cobra.Command{
 }
 
 func Execute() {
-	rootCmd.Execute()
+	if err := rootCmd.Execute(); err != nil {
+		log.Fatal(err)
+		os.Exit(-1)
+	}
+}
+
+func init() {
+	cobra.OnInitialize(initConfig)
+}
+
+func initConfig() {
+	// set up ENV var support
+	viper.SetEnvPrefix("pflt")
+	viper.AutomaticEnv()
+
+	// set up optional config file support
+	viper.SetConfigName("config")
+	viper.SetConfigType("yaml")
+	viper.AddConfigPath(".")
+	if err := viper.ReadInConfig(); err != nil {
+		if _, ok := err.(viper.ConfigFileNotFoundError); ok {
+			log.Info("Config file not found.  Proceeding without it.")
+		}
+	}
+
+	// Set up logging config defaults
+	viper.SetDefault("logfile", "preflight.log")
+	viper.SetDefault("loglevel", "warn")
```

Comment:
_see review-level comment_

This also contains a hard-coded default `preflight.log` that probably should have made reference to our globally defined constant. Same for the log level, but we didn't have that configurable defined at all previously.

Reply:
Utilize variables defined in constants.go for the default value of loglevel and logfile so we don't hard-code it here.